source/draw/gpu/opengl/EGLYuv.ooc

```diff
@@ -23,5 +23,10 @@ EGLYuv: class extends OpenGLPacked {
 	}
 	toRasterDefault: override func -> RasterImage { Debug error("toRasterDefault unimplemented for EGLYuv"); null }
 	toRasterDefault: override func ~target (target: RasterImage) { Debug error("toRasterDefault~target unimplemented for EGLYuv") }
+	draw: override func ~DrawState (drawState: DrawState) {
+		//Blending is not supported for this image type, so we force it off
```

Comment:
It would be more consistent with other comments by using a space after `//`.
But it seems like 95% of all comments are written by me.